test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -216,7 +216,7 @@ BOOST_AUTO_TEST_CASE(smoke_test)
 	char const* text = R"(
 		contract test {
 			uint256 stateVariable1;
-			function fun(uint256 arg1) { uint256 y; }
+			function fun(uint256) { uint256 y; }
```

Comment:
I will.  By the way, the reason the test passes is because there is only 1 warning, so the "multiple errors" check doesn't fail, and since the only reported "error" is a warning, it doesn't count as an error on the final check.